src/rules/use-tokens.test.js

```diff
@@ -181,20 +181,21 @@ const styles = StyleSheet.create({
       code: `
 const styles = StyleSheet.create({
   foo: {
-    color: 'rgb(0, 178, 214)',
+    color: 'rgb(7, 112, 227)',
   },
 });`,
       output: `
-import { colorBlue500 } from 'bpk-tokens/tokens/base.es6';
+import { colorPrimaryGradientLight } from 'bpk-tokens/tokens/base.es6';
```

Comment:
This has had to be used over `colorSkyBlue` as it defines the same colour but as its defined after SkyBlue its picked up in the test instead and causes it to fail.

TODO: This should be changed once this token has been removed from Backpack tokens